src/common/utils/hooks.tsx

```diff
@@ -1,8 +1,8 @@
 import { useLocation, useParams } from 'react-router-dom';
 
 // TODO: Mv to enums?
-const EMBED_WIDTH = 350;
-const EMBED_HEIGHT = 700;
+const EMBED_WIDTH = 375;
+const EMBED_HEIGHT = 393;
 
```

Comment:
Would check the js snippet to ensure that we're using the same dimensions when we dynamically create

Reply:
Yeah, a quick test suggests something is off.

![image](https://user-images.githubusercontent.com/206364/83803088-cb884c80-a660-11ea-868b-2539647adb1d.png)

Reply:
Ooh yes! Fixed in embed.js